Build error on windows_amd64 binary

https://circleci.com/gh/int128/kubelogin/314

```
VERSION=v1.12.0 goxzst -d dist/gh/ -o "kubelogin" -t "kubelogin.rb oidc-login.yaml" -- -ldflags "-X main.version=v1.12.0"
2019/06/05 13:04:42 GOOS=linux GOARCH=amd64 go build -o dist/gh/kubelogin_linux_amd64 -ldflags -X main.version=v1.12.0
2019/06/05 13:04:43 Creating dist/gh/kubelogin_linux_amd64.zip
2019/06/05 13:04:44 Creating dist/gh/kubelogin_linux_amd64.zip.sha256
2019/06/05 13:04:44 GOOS=darwin GOARCH=amd64 go build -o dist/gh/kubelogin_darwin_amd64 -ldflags -X main.version=v1.12.0
2019/06/05 13:04:52 Creating dist/gh/kubelogin_darwin_amd64.zip
2019/06/05 13:04:53 Creating dist/gh/kubelogin_darwin_amd64.zip.sha256
2019/06/05 13:04:53 GOOS=windows GOARCH=amd64 go build -o dist/gh/kubelogin_windows_amd64.exe -ldflags -X main.version=v1.12.0
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: extracting github.com/inconshreveable/mousetrap v1.0.0
# github.com/int128/kubelogin/adaptors/env
adaptors/env/env.go:22:33: cannot use syscall.Stdin (type syscall.Handle) as type int in argument to terminal.ReadPassword
2019/06/05 13:05:01 Error: error while build for the platform windows_amd64: error while cross build: go build error: error while exec: exit status 2
```

--- a/adaptors/env/env.go
+++ b/adaptors/env/env.go
@@ -19,7 +19,7 @@ func (*Env) ReadPassword(prompt string) (string, error) {
 	if _, err := fmt.Fprint(os.Stderr, "Password: "); err != nil {
 		return "", errors.Wrapf(err, "could not write the prompt")
 	}
-	b, err := terminal.ReadPassword(syscall.Stdin)
+	b, err := terminal.ReadPassword(int(syscall.Stdin))
 	if err != nil {
 		return "", errors.Wrapf(err, "could not read")
 	}

